### PR TITLE
Edit profile basic

### PR DESCRIPTION
**Pull request by PhamVanTung**
This PR for ticket https://dev.framgia.com/redmine/issues/60869
Please check it.
Thank

### CODE REVIEW

File: app/scripts/controllers/users_controller.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  UsersController.$inject = ['$scope', '$location', 'DataFactory', '$firebaseObject'];
+
+  function UsersController($scope, $location, DataFactory, $firebaseObject){
+    var user = this;
+    var key = 'Anh A';
+    var usersRef = new DataFactory('users');
+
+    user.info = $firebaseObject(usersRef.child(key));
+
+    $scope.saveProfile = function(){
```
Comment (xuanduc987) [inline app/scripts/controllers/users_controller.js]: `user.saveProfile = ...`

File: app/scripts/controllers/users_controller.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+(function(){
+  'use strict';
+
+  angular
+    .module('app.controllers')
+    .controller('UsersController', UsersController);
+
+  UsersController.$inject = ['$scope', '$location', 'DataFactory', '$firebaseObject'];
```
Comment (xuanduc987) [inline app/scripts/controllers/users_controller.js]: `$scope, $location` is not being used

Actually, the spec said it need to redirect to `profile` page, but for now just redirect to `main`

File: app/scripts/controllers/users_controller.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  angular
+    .module('app.controllers')
+    .controller('UsersController', UsersController);
+
+  UsersController.$inject = ['$location', 'DataFactory', '$firebaseObject'];
+
+  function UsersController($location, DataFactory, $firebaseObject){
+    var user = this;
+    var key = 'Anh A';
```
Comment (yeuem1vannam) [inline app/scripts/controllers/users_controller.js:14]: Leave a comment for other ppl can understand that this is just stubbed data

File: app/scripts/controllers/users_controller.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    var usersRef = new DataFactory('users');
```
Comment (yeuem1vannam) [inline app/scripts/controllers/users_controller.js]: Don't use `new` for `DataFactory`

**Comment by yeuem1vannam** [comment]
:o:K

**Pull request merged**